client/my-sites/plugins/controller-logged-out.js

```diff
@@ -0,0 +1,73 @@
+import {
+	getWPCOMFeaturedPluginsQueryParams,
+	getWPCOMPluginsQueryParams,
+} from 'calypso/data/marketplace/use-wpcom-plugins-query';
+import { getWPORGPluginsQueryParams } from 'calypso/data/marketplace/use-wporg-plugin-query';
+import { requestProductsList } from 'calypso/state/products-list/actions';
+import { getProductsList } from 'calypso/state/products-list/selectors';
+
+function getProps( context ) {
+	const props = {
+		path: context.path,
+		locale: context.lang,
+		tag: '',
+	};
+	return props;
```

Comment:
Nit: this could be shortened: 

```suggestion
function getProps( { path, lang } ) {
	return {
		path,
		locale: lang,
		tag: '',
	};
```